src/styles/theme.css

```diff
@@ -70,3 +70,15 @@ app-header {
   display: block;
   margin: 10px 0 0;
 }
+
+h2 > a > span.icon {
```

Comment:
for deep link headings, convention seems to be
1. Show on hover
1. (optionally) move it to the right
1. freshen up the link SVG we're using?